### DISCUSSION TITLE
Volunteer Lead Gen / Onboarding Automation

### DISCUSSION THREAD

**Discussion by profetik-777** [danielmiessler/Fabric#137] [Ideas]
As per feedback from Daniel dropping this in Discussions( https://github.com/danielmiessler/fabric/issues/38#issuecomment-1928513674 ) , pitching an idea for potential area of contribution.

GOAL:

Help create a simple and easy to manage developer volunteer pipeline so Daniel has a decent volunteer "roster" to lean on to solve small to mid-level issues (or feature enhancements).

Details:

    Use a website asset (or online form) that Daniel can share from time to time (or put against ads on social media) with a "Call for Volunteers" that quickly and easily outlines the following:

    preferred languages
    areas of need
    commitment level
    contact info (in case there isn't an immediate need- but you want to reach out to them later).

This way - Daniel can manage expectations of those who fill out the form, while also quickly size up basic info about their background. You can even feature in the form - an example of a great contribution/ pull request so users can align their expectations to what you need.

What is needed:

    A place to submit sample form questions (so Daniel can finalize/approve). Do this in a pull request? Or just send a you a doc?
    Have Daniel list out preferred/ideal languages/experience that would be a good fit.
    A sample pull request he likes so it can be featured in the form
    BONUS/ STRETCH GOAL: Attach this form (or link if the form lives in your website) to a BOT in your chats in discord or wherever your community lives, and have it get shared 1/month or whatever frequency. This way - you don't have to think about "recruiting" contributors. It does it for you.....
    Bonus Bonus ;) You could experiment with running a small targeted add, to really see where recruiting volunteers can take you. I have some digital marketing experience that can assist with that - but that might be overkill. But maybe not, if you are interested in building a solid core team that you can actually trust.

**Comment by danielmiessler** [comment]
I think this is a really cool idea. But the default for Github might just be people doing the work and submitting PRs. Maybe that's just how things are normally done?

**Comment by profetik-777** [comment]
@danielmiessler sorry you feel that way. 

The problem this solves is assist with the speed of development, while at the same time, getting assistance on those odd one off issues/fixes or enhancements that tend to live in backlog longer than we like. 

One thing that holds amazing foss software back is product polish...having a healthy pipeline of volunteers to target a "themed" sprint or two might have assisted with this can have an incredible impact on user adoption (aka more people who get empowered by your AI tools).

At the end of the day - I'm interested in ensuring lead dev avoids burnout (I hate when awesome projects start off with a bang then fizzle out) and users get empowered via FOSS (instead of major corporations building products and benefiting from FOSS development and repackaging it at cost or quick and dirty UX overlay). 

I'm curious - because you are not the first dev to have initially love the idea to only shy away from it - what would have made you say "yes"? 

Maybe the next project I help with - I can better pitch the vision and persuade of the importance/value?
